cdap-app-fabric/src/main/java/co/cask/cdap/metadata/LineageAdmin.java

```diff
@@ -120,41 +104,10 @@ public Lineage computeLineage(final DatasetId sourceDataset, long startMillis, l
    * @param levels number of levels to compute lineage for
    * @return lineage for sourceDataset
    */
-  public Lineage computeLineage(final DatasetId sourceDataset, long startMillis, long endMillis,
-                                int levels) throws NotFoundException {
+  public Lineage computeLineage(final DatasetId sourceDataset, long startMillis, long endMillis, int levels) {
     return doComputeLineage(sourceDataset, startMillis, endMillis, levels, null);
   }
 
-  /**
-   * @return metadata associated with a run
-   */
-  public Set<MetadataRecord> getMetadataForRun(ProgramRunId run) {
```

Comment:
Is this removed because it is not being used anywhere?

Reply:
correct. It was decided in 5.x to deprecate it and in 6.0 we removed support for this. The UI never used it anyway.